webapp/loggedin/modules/data/records/recordsView.js

```diff
@@ -56,7 +64,8 @@ const mapStateToProps = state => ({
   lang: AppState.getLang(state),
   user: AppState.getUser(state),
   surveyInfo: SurveyState.getSurveyInfo(state),
-  nodeDefKeys: RecordsState.getNodeDefKeys(state),
+  surveyCycleKey: SurveyState.getSurveyCycleKey(state),
```

Comment:
why are you passing surveyCycleKey to TableView?

Reply:
it's used to detect a change of the survey cycle